app/assets/src/components/views/TaxonTreeVis.jsx

```diff
@@ -0,0 +1,296 @@
+import React from "react";
+import PropTypes from "prop-types";
+import TidyTree from "../visualizations/TidyTree";
+import PathogenLabel from "../ui/labels/PathogenLabel";
+
+const TaxonLevels = [
+  "species",
+  "genus",
+  "family",
+  "order",
+  "class",
+  "phylum",
+  "kingdom",
+  "superkingdom"
+];
+class TaxonTreeVis extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      nodeHover: null
+    };
+
+    this.nameType = this.props.nameType;
+    this.metric = this.props.metric || "aggregatescore";
+    this.taxa = this.props.taxa;
+
+    this.tree = null;
+    this.treeVis = null;
+
+    this.metrics = {
+      aggregatescore: {
+        label: "Aggregate Score",
+        agg: arr => Math.max(...arr)
+      },
+      nt_r: { label: "NT - r", agg: arr => arr.reduce((a, b) => a + b, 0) },
```

Comment:
can we use something other than "-"? It looks like "minus"

Reply:
Sure. What do you suggest?

Reply:
I'd have suggested no delimiter at all, since that's how it is for the threshold filters in the report

Reply:
Sure, will remove.